File: app/src/main/java/life/plank/juna/zone/data/network/model/firebaseModel/Thumbnail.java
package life.plank.juna.zone.data.network.model.firebaseModel;

public class Thumbnail {

    private String imageUrl;
    private Integer height;
    private Integer width;

    public Thumbnail(String imageUrl, Integer height, Integer width) {
        this.imageUrl = imageUrl;
        this.height = height;
        this.width = width;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public Integer getHeight() {
        return height;
    }

    public void setHeight(Integer height) {
        this.height = height;
    }

    public Integer getWidth() {
        return width;
    }

    public void setWidth(Integer width) {
        this.width = width;
    }
}

File: app/src/main/java/life/plank/juna/zone/data/model/feed/Thumbnail.kt
package life.plank.juna.zone.data.model.feed

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Thumbnail(
        var imageUrl: String = "",
        var imageHeight: Int = 0,
        var imageWidth: Int = 0
) : Parcelable